client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -452,7 +464,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 
 	function unlockPremiumGlobalStyles() {
 		// These conditions should be true at this point, but just in case...
-		if ( selectedDesign && numOfSelectedGlobalStyles ) {
+		if ( shouldUnlockGlobalStyles ) {
```

Comment:
I will review tomorrow but first I want to confirm if the comment above this line is still true? 😄 

```
// These conditions should be true at this point, but just in case...
```

Reply:
(Because we modified the boolean condition)

Reply:
Yes, I think so